libfwupdplugin/fu-common.c

```diff
@@ -2256,7 +2256,8 @@ fu_byte_array_set_size (GByteArray *array, guint length)
 gboolean
 fu_common_kernel_locked_down (void)
 {
-#ifndef _WIN32
+#if !defined(_WIN32) && !defined(__FreeBSD__) && !defined(__DragonflyBSD__) \
```

Comment:
If these will all be the same for WIN32 and BSD I wonder if it's worth making a meson.build level define for this kind of thing?

Reply:
There are some WIN32 specific lines, that are not related to BSD distros, but most of the WIN32 ifdefs apply to BSD as well.

Reply:
Can we just do `ifdef __linux__` or something? That's basically what we're asking...

Reply:
Sure, it will be more readable.